test/functional/wallet_showconflictedinactivetransactions.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test that listtransactions shows whether a transaction is in the mempool."""
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+
+class WalletShowConflictedInactiveTransactionsTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 1
+
+    def skip_test_if_missing_module(self):
+        self.skip_if_no_wallet()
+
+    def create_and_send_transaction(self, utxo, address, amt, feeRate):
+        psbtx = self.nodes[0].walletcreatefundedpsbt([{"txid": utxo['txid'], "vout": utxo['vout']}],
+                                                      {address: amt},
+                                                      0,
+                                                      {"replaceable":True, "feeRate":feeRate})['psbt']
+        signed_tx = self.nodes[0].walletprocesspsbt(psbtx)['psbt']
+        final_tx = self.nodes[0].finalizepsbt(signed_tx)['hex']
+        return self.nodes[0].sendrawtransaction(final_tx)
+
+    def test_listtransactions_display_inmempool(self):
+        utxo = self.nodes[0].listunspent()[0]
+        address = self.nodes[0].getnewaddress()
+
+        tx1_id = self.create_and_send_transaction(utxo, address, 10, 0.001)
+
+        new_txs = self.nodes[0].listtransactions(count=2)
```

Comment:
Where `listtransactions()#txid/in_mempool` are tested, could add a regression test for `gettransaction()` as well.